icu4c/source/common/unicode/uvernum.h

```diff
@@ -106,13 +106,21 @@
 
 #ifndef U_ICU_ENTRY_POINT_RENAME
 #ifdef U_HAVE_LIB_SUFFIX
-#define U_DEF_ICU_ENTRY_POINT_RENAME(x,y,z) x ## y ##  z
-#define U_DEF2_ICU_ENTRY_POINT_RENAME(x,y,z) U_DEF_ICU_ENTRY_POINT_RENAME(x,y,z)
-#define U_ICU_ENTRY_POINT_RENAME(x)    U_DEF2_ICU_ENTRY_POINT_RENAME(x,U_ICU_VERSION_SUFFIX,U_LIB_SUFFIX_C_NAME)
+  #if !U_DISABLE_VERSION_SUFFIX
```

Comment:
Please use customary indentation style: The hash symbol at the start of the line, followed by 3, 7, 11, ... spaces.

Reply:
Please also add a javadoc section
```
/**
  * Explain this here?
  * \def U_DISABLE_VERSION_SUFFIX
 * @internal
 */
```